lib/pages/labels/label_bloc.dart

```diff
@@ -47,13 +47,23 @@ class LabelBloc implements BlocBase {
     _loadLabels();
   }
 
-  void checkIfLabelExist(Label label) async {
-    _labelDB.isLabelExits(label).then((isExist) {
-      _labelExistController.sink.add(isExist);
+  void createOrExists(Label label) async {
```

Comment:
Can you please write unit tests for this?

Reply:
Test case : `"Add label if not exist in the label db test"` and `"Don't Add label if exist in the label db test"` are covering this functionality.